src/tutorials/0004-mutable-file-system/08.js

```diff
@@ -120,7 +120,7 @@ const run = async (files) => {
   const rootDirectoryContents = await all(ipfs.files.ls('/'))
 
   const filepathsToMove = rootDirectoryContents.filter(file => file.type === 0).map(file => '/' + file.name)
-  await ipfs.files.mv(filepathsToMove, '/some/stuff')
+  await ipfs.files.mv(...filepathsToMove, '/some/stuff')
```

Comment:
The spread syntax feels pretty new and is probably worth an explanation or a hint w/ link to docs.